Makefile

```diff
@@ -139,6 +139,9 @@ e2e-test-jobseq:
 e2e-test-vcctl:
 	E2E_TYPE=VCCTL ./hack/run-e2e-kind.sh
 
+e2e-test-stress:
```

Comment:
Suggest to give another test script. Stress test should be an option test and not be inclued into CI

Reply:
@Thor-wl Thanks for the review! I suppose the build target defined in `Makefile` would not influence the GitHub Action workflows. Currently, the triggered workflows are *Basic Scheduling*, *Parallel Jobs*, *Scheduling Actions*, *Scheduling Basic*, and *Vcctl*.

Since stress tests are parts of e2e tests, they also have to be executed by KinD and Ginkgo.

Reply:
OK. THX.